worlds/civ_6/Container.py

```diff
@@ -0,0 +1,219 @@
+from dataclasses import dataclass
+import os
+from typing import TYPE_CHECKING, Dict, List, Optional, cast
+import zipfile
+from BaseClasses import Location
+from worlds.Files import APContainer
+
+from .Enum import CivVICheckType
+from .Locations import CivVILocation, CivVILocationData
+
+if TYPE_CHECKING:
+    from worlds.civ_6 import CivVIWorld

```

Comment:
Own-world imports must be relative
```suggestion
    from . import CivVIWorld
```